docs-user/guide-remote-profiling.md

```diff
@@ -0,0 +1,14 @@
+# Remote profiling on Android
```

Comment:
What do you think about breaking it down further?

To section this off better, maybe

```
Explainer (set expectations, list compatible platforms)
# Setup
## Enable new Profiler panel
## Enable remote debugging on mobile device
# Recording
# Tips
- Only open the Performance panel, to reduce the overhead of activated panels.
```

Reply:
I like it. As long as it doesn't end up with single-sentence paragraphs!